runner/file.go

```diff
@@ -536,6 +569,7 @@ const (
 	OpenOfficeSheetMimeType          = "application/vnd.oasis.opendocument.spreadsheet"
 	ParquetMimeType                  = "parquet"
 	ORCMimeType                      = "orc"
+	AvroMimeType                     = "avro"
```

Comment:
This one has a real mimetype unlike Parquet and ORC. It should be `application/avro`.